djstripe/models.py

```diff
@@ -106,7 +124,7 @@ def __str__(self):
 @python_2_unicode_compatible
 class Event(StripeObject):
     stripe_api_name = "Event"
-
+    objects = models.Manager()
```

Comment:
Interesting read. According to that section, moving ``objects`` into the superclass and before ``stripe_objects`` works: [patch](https://gist.github.com/kavdev/c5dc8d0ba79142354151)